File: AccountBalanceAPI/AccountBalanceAPI/Model/AccountBalance.cs
using System;
using System.ComponentModel.DataAnnotations;

namespace AccountBalanceAPI.Model
{
    public class AccountBalance
    {
        [Key]
        public int BalanceID { get; set; }
        public DateTime BalanceDate { get; set; }
        public decimal BalanceRAD { get; set; }
        public decimal BalanceCanteen { get; set; }
        public decimal BalanceCEOCar { get; set; }
        public decimal BalanceMarketing { get; set; }
        public decimal BalanceParkingFines { get; set; }
    }
}

File: AccountBalanceApp/src/app/Model/accountbalance.ts
export class AccountBalance {
    constructor(
        public BalanceID: number,
        public BalanceDate: Date,
        public BalanceRAD: number,
        public BalanceCanteen: number,
        public BalanceCEOCar: number,
        public BalanceMarketing: number,
        public BalanceParkingFines: number,
    ) { }
}